ui/src/components/PieChart/PieChart.css

```
@@ -0,0 +1,4 @@
+#pie-chart-wrapper {
```

Wouldn't this be a class selector rather than an id selector? I don't see why `id` is necessary here for the div it affects:
```suggestion
.pie-chart-wrapper {
```

This also requires the div in `PieChart.jsx` to have its property changed as well.